utils/use_require.js

```diff
@@ -231,7 +235,29 @@ var make_lib_files = function (import_format, source_maps, with_app_dir) {
         const out_app_path = path.join(out_path_base, 'app.js');
         console.log(`Writing ${out_app_path}`);
         return helper.appWriter(out_path_base, out_app_path)
-        .then(transform_html);
+        .then(transform_html)
+        .then(() => {
+            if (!helper.removeModules) return;
+            console.log(`Cleaning up temporary files...`);
+            let unlink = path => new Promise((resolve, reject) => {
+                fse.unlink(path, err => {
+                    if (err) return reject(err);
+                    resolve();
+                });
+            });
+            let rmdir_r = dir => new Promise((resolve, reject) => {
+                fse.rmdir(dir, err => {
```

Comment:
https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md

Use this for recursive removal

Reply:
I only want to remove empty directories, so that function would not work.